src/content/docs/blog/biome-v2-0-beta.md

```diff
@@ -0,0 +1,177 @@
+
+The HTML formatter is still pretty experimental, so it will remain **disabled by default for the full 2.0 release**. At the time of writing, Biome is able to parse the grand majority of Prettier's HTML tests, and format 46/124 of them correctly. Despite not matching Prettier yet, we're pretty confident that it _should_ output documents that are formatted adequately without destroying anything. If you find a case where it doesn't, [please let us know](https://github.com/biomejs/biome/issues)!
+
+You can enable the HTML formatter by adding the following to your config file:
+
+```json
+{
+  "html": {
+    "formatter": {
+      "enabled": true
+    }
+  }
+}
+```
+
+### New rules
+
+Several new rules have added since v1.9:
+
+- [`noAwaitInLoop`](https://biomejs.dev/linter/rules/no-await-in-loop)
+- [`noBitwiseOperators`](https://biomejs.dev/linter/rules/no-bitwise-operators/)
+- [`noDestructuredProps`](https://biomejs.dev/linter/rules/no-destructured-props/)
+- [`noFloatingPromises`](https://biomejs.dev/linter/rules/no-floating-promises)
+- [`noImportCycles`](https://biomejs.dev/linter/rules/no-import-cycles)
+- [`noTsIgnore`](https://biomejs.dev/linter/rules/no-ts-ignore)
+- [`noUnwantedPolyfillio`](https://biomejs.dev/linter/rules/no-unwanted-polyfillio)
+- [`useConsistentObjectDefinition`](https://biomejs.dev/linter/rules/use-consistent-object-definition/)
+- [`useForComponent`](https://biomejs.dev/linter/rules/use-for-component/)
+
+### Miscellaneous
+
+- **BREAKING:** The configuration fields `include` and `ignore` have been replaced with a single `includes` field.
+- **BREAKING:** Reworked some recommended rules recommended to be less pedantic and blocking. This is a breaking change if your project relied on those rules to block the CI in case of violations. If you used the `migrate` command, the behaviour should remain as before.
+- **BREAKING:** The `style` rules aren't recommended anymore. If you used the `migrate` command, the behaviour should remain as before.
+- **BREAKING:** Cleaned up deprecated rules:
+  - `noConsoleLog`
+  - `noInvalidNewBuiltin`
+  - `noNewSymbol`
+  - `useShorthandArrayType`
+  - `useSingleCaseStatement`
+- **BREAKING:** Many deprecated options, including some from the Rome days, have been removed.
+- Added a new option `javascript.parser.jsxEverywhere` to control whether Biome should expect JSX syntax in `.js`/`.mjs`/`.cjs` files.
+- Improved monorepo support: The rule [`noUndeclaredDependencies`](https://biomejs.dev/linter/rules/no-undeclared-dependencies/) now works correctly in monorepos by using the nearest `package.json` file, instead of only the root one.
+- We have enabled support for `.editorconfig` files by default.
+- Changed default formatting of `package.json` to align better with formatting by package managers.
+
+### And more!
+
+For the full list of changes, please refer to our [changelog](/internals/changelog/).
```

Comment:
Unless we are able to provide a changelog for the beta, we should remove it. We still don't know yet **how** we will generate the beta, and the changelog creation is now owned by changesets

Reply:
Yeah, I had assumed we could create a new changelog from the changesets, but let's see how that works out :sweat_smile:

Reply:
It is possible, but we need to change the script that does that